script/Deploy.s.sol

```diff
@@ -1,9 +1,23 @@
-// SPDX-License-Identifier: UNLICENSED
-
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
*(arbitrary comment)*

- This is pretty much the same as gitcoin

Reply:
Let's include updates to the README explaining how to use the script in the PR, similar to GC

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/6/commits/7f44d8e4219d164eb2ebb130aadd53a3b50a4c81